.github/workflows/destroy.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     # This conditional is a backup mechanism to help prevent mistakes from becoming disasters.
     # This is a list of branch names that are commonly used for protected branches/environments.
     # Add/remove names from this list as appropriate.
-    if: github.event.ref_type == 'branch' && !contains(fromJson('["develop", "main", "master", "impl", "val", "prod", "production"]'), github.event.ref)
+    if: github.event.ref_type == 'branch' && !contains(fromJson('["master", "val", "production"]'), github.event.ref)
```

Comment:
We only use master, val, and production for higher environments.  These additional branches were added to help forks, but it is difficult to explain, bloats our code, and does not encourage awareness.